xml/cap_admin_app_autoscaler.xml

```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.app-autoscaler"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>App-AutoScaler</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  The App-AutoScaler service is used for automatically managing an application's instance count when deployed on &scf;. The scaling behavior is determined by a set of criteria defined in a policy (See <xref linkend="sec.cap.app-autoscaler-policies"/>).
+ </para>
+
+ <sect1 xml:id="sec.cap.app-autoscaler-prereqs">
+  <title>Prerequisites</title>
+  <para>
+   Using the App-AutoScaler service requires:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     <link xlink:href="https://docs.cloudfoundry.org/cf-cli/">The &cf; command line interface (&cfcli;)</link>
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <link xlink:href="https://github.com/cloudfoundry-incubator/app-autoscaler-cli-plugin">The App-AutoScaler &cfcli; Plugin</link>
+    </para>
+    <para>
+     The plugin can be installed by running the following command:
+    </para>
+<screen>cf install-plugin app-autoscaler-plugin</screen>
+   </listitem>
+   <listitem>
+    <para>
+     Review of the <link xlink:href="https://github.com/cloudfoundry-incubator/app-autoscaler/tree/master/docs#things-you-need-to-know-before-auto-scale-an-application">considerations and guidelines</link>.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+
+ <sect1 xml:id="sec.cap.enable-app-autoscaler">
+  <title>Enabling the App-AutoScaler Service</title>
+  <para>
+   By default, the App-AutoScaler service is not enabled as part of a &scf; deployment. To enable it, add the following values to your <filename>scf-config-values.yaml</filename> file and deploy SCF:
+  </para>
+<screen>
+sizing:
+  autoscaler_actors:
+    count: 1
+  autoscaler_api:
+    count: 1
+  autoscaler_metrics:
+    count: 1
+  autoscaler_postgres:
+    count: 1
+</screen>
+  <para>
+   See <command>helm inspect suse/cf</command> for the full list of available values that can be used to configure the App-AutoScaler.
```

Comment:
Does the Perl example in http://docserv.nue.suse.com/documents/CAP_1/cap-guides/single-html/#sec.cap.ha-defaults extract the necessary information for this?

Reply:
Yep (see below), but the `see helm inspect suse/cf` is intended more for the other values available (which I'm not e.g. specify certs, timeouts, TTLs, etc) and not just enabling it.

```
helm inspect values suse/cf | \
perl -ne '/^sizing/..0 and do { print $.,":",$_ if /^ [a-z]/ || /high avail|scale|count/ }'
1049:    # The adapter instance group can scale between 1 and 65535 instances.
1050:    # For high availability it needs at least 2 instances.
1051:    count: 1
1091:    # The api-group instance group can scale between 1 and 65535 instances.
1092:    # For high availability it needs at least 2 instances.
1093:    count: 1
1105:  # The autoscaler-actors instance group contains the following jobs:
1113:  autoscaler_actors:
1120:    # The autoscaler-actors instance group can scale between 0 and 3 instances.
1121:    # For high availability it needs at least 2 instances.
1122:    count: 0
1134:  # The autoscaler-api instance group contains the following jobs:
1142:  autoscaler_api:
1149:    # The autoscaler-api instance group can scale between 0 and 3 instances.
1150:    # For high availability it needs at least 2 instances.
1151:    count: 0
1163:  # The autoscaler-metrics instance group contains the following jobs:
1171:  autoscaler_metrics:
1178:    # The autoscaler-metrics instance group can scale between 0 and 3 instances.
1179:    # For high availability it needs at least 2 instances.
1180:    count: 0
1192:  # The autoscaler-postgres instance group contains the following jobs:
1200:  autoscaler_postgres:
1207:    # The autoscaler-postgres instance group can scale between 0 and 3
1209:    # For high availability it needs at least 2 instances.
1210:    count: 0
```

Reply:
ok!